src/convertToObject.js

```diff
@@ -7,6 +7,20 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .trim()
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style)
+    .reduce((stylesObject, style) => {
+      const [property, value] = style.split(':').map((part) => part.trim());
+
+      if (property && value) {
+        stylesObject[property] = value;
+      }
+
+      return stylesObject;
+    }, {});
```

Comment:
The chained methods have the same indentation, which is good for readability. However, ensure that any additional chained methods follow the same indentation pattern.